### PR TITLE
feat: implement section and columns

### PR DESCRIPTION
**Pull request by arikchakma**
## Features

- Section with padding, border radius, and background color
- 2 ways Columns

### CODE REVIEW

File: packages/core/src/editor/components/section-menu/section-bubble-menu.tsx [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+      <NumberInput
+        icon={Scan}
+        value={state.currentBorderRadius}
+        onValueChange={(value) => {
+          editor?.commands?.updateSection({
+            borderRadius: value,
+          });
+        }}
+      />
+      <div className="mly-mx-0.5 mly-w-px mly-bg-gray-200" />
```
Comment (SokratisVidros) [inline packages/core/src/editor/components/section-menu/section-bubble-menu.tsx:71]: Nit picky one: You can set this as a `const divider = ...` in this function.

File: packages/core/src/editor/components/ui/color-picker.tsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+  const handleColorChange = (color: string) => {
+    // HACK: This is a workaround for a bug in tiptap
+    // https://github.com/ueberdosis/tiptap/issues/3580
+    //
+    //     ERROR: flushSync was called from inside a lifecycle
+    //
+    // To fix this, we need to make sure that the onChange
+    // callback is run after the current execution context.
+    queueMicrotask(() => {
```
Comment (SokratisVidros) [inline packages/core/src/editor/components/ui/color-picker.tsx:23]: 😍

File: packages/core/src/editor/extensions/slash-command.tsx [hunk 3]
```diff
@@ -247,6 +249,30 @@ const DEFAULT_SLASH_COMMANDS: SlashCommandItem[] = [
       editor.chain().focus().selectParentNode().deleteSelection().run();
     },
   },
+  {
+    title: 'Columns',
+    description: 'Add columns to email.',
```
Comment (SokratisVidros) [inline packages/core/src/editor/extensions/slash-command.tsx:254]: It's worth making this copywriting configurable at some point in the future.

**Review by SokratisVidros — APPROVED** [review]

File: packages/core/src/editor/components/ui/color-picker.tsx (hunk 2 above)
Comment (arikchakma) [reply packages/core/src/editor/components/ui/color-picker.tsx:23]: I didn't get this one

File: packages/core/src/editor/components/section-menu/section-bubble-menu.tsx (hunk 1 above)
Comment (arikchakma) [reply packages/core/src/editor/components/section-menu/section-bubble-menu.tsx:71]: Sure, I will make a component maybe. Let me do it.

File: packages/core/src/editor/components/ui/color-picker.tsx (hunk 2 above)
Comment (SokratisVidros) [reply packages/core/src/editor/components/ui/color-picker.tsx:23]: I just liked the hack ;)

Comment (arikchakma) [reply packages/core/src/editor/components/ui/color-picker.tsx:23]: Haha, thanks😆

**Pull request merged**